homework/index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html>
+
+    <head>
+
+        <meta charset="UTF-8">
+        <title>HUDA ALISMAIL</title>
+        <link rel="stylesheet" href ="styles/styles.css">
+
```

Comment:
be careful the head tag should only contain things that you don't want to show in the browser.